AdornUI-Stylesheets/badges.css

```diff
@@ -0,0 +1,105 @@
+h1 {
+  font-size: 32px;
+}
+h2 {
+  font-size: 24px;
+}
+h3 {
+  font-size: 18.72px;
+}
+h4 {
+  font-size: 16px;
+}
+h5 {
+  font-size: 12px;
+}
+.badge {
+  padding: 0.2rem 0.7rem;
+  margin: 0.5rem;
+  border-radius: 5px;
+  font-size: 0.75em;
+}
+.badge-primary {
+  background-color: var(--primary-bg-color);
+  color: var(--white);
+}
+.badge-secondary {
+  background-color: var(--secondary-text-color);
+  color: var(--white);
+}
+.badge-round-pill {
+  border-radius: 50em;
+  background-color: var(--secondary-text-color);
+  color: var(--white);
+}
+.badge-w-icon-container {
+  display: flex;
+  flex-direction: row;
+  gap: 3rem;
+  padding: 2rem;
+}
+.fa-shopping-cart,
+.fa-heart,
+.fa-comment {
+  font-size: 5rem;
+  position: relative;
+  color: var(--secondary-text-color);
+}
```

Comment:
fa-shopping-cart , fa-heart , fa-comment are not easily understandable . Use a name that is easily understandable.